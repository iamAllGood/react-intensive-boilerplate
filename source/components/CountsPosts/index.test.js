import dom from 'react-test-renderer';
import Counter from './';

const renderTree = dom.create(<Counter countsPosts = { 3 } />).toJSON();

test('Counter should correspound to its snapshot counterpart', () => {
    expect(renderTree).toMatchSnapshot();
});
