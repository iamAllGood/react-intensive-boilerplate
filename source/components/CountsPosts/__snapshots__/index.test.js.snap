// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should correspound to its snapshot counterpart 1`] = `
<div>
  undefined: 3 posts. 
</div>
`;
